### PR TITLE
enabling Windows port.

### PR DESCRIPTION
**Pull request by seanyen**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
@conda-forge/jbig This is ready for review and merge. Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -5,20 +5,23 @@ package:
...
-  number: 2001
-  skip: true  # [win]
+  number: 2002
+  skip: true  # [win and vc<14]
 
 requirements:
   build:
-    - {{ compiler('c') }}
+    - {{ compiler('c') }}  # [not win]
+    - vs2017_win-64        # [win]
```
Comment (nehaljwani) [inline recipe/meta.yaml]: Why is this necessary?

Comment (seanyen) [reply recipe/meta.yaml]: @nehaljwani By default, it picks up the vs2015 on Windows. However somehow I just cannot get it built on the cloud pipeline and a mysterious error throw to me (see below). So I gave it a try to the newer toolchain and the problem is just gone.

```
[100%] Linking C shared library libjbig.dll
NMAKE : fatal error U1077: 'D:\bld\jbig_1585696984760\_build_env\Library\bin\cmake.exe' : return code '0xc0000005'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\nmake.exe"' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\nmake.exe"' : return code '0x2'
Stop.

```

**Comment by seanyen** [comment]
@conda-forge/jbig @conda-forge/staged-recipes Friendly ping. This is ready for review and merge. Thanks.

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/meta.yaml]: You can remove this change and rerender in an hour. See https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/501

Comment (seanyen) [reply recipe/meta.yaml]: @isuruf Hooray! And just a quick question, as the default is going to be migrated to vs2017, is there a plan to start adding `vs2019_win-64` (on conda-forge) for preview.

Comment (isuruf) [reply recipe/meta.yaml]: PRs welcome

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

File: recipe/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "libjbig/jbig_ar.c"
+    "libjbig/jbig85.c")
+
+set(HEADERS
+    "libjbig/jbig.h"
+    "libjbig/jbig_ar.h"
+    "libjbig/jbig85.h"
+)
+
+add_library(libjbig ${SOURCES})
```
Comment (isuruf) [inline recipe/CMakeLists.txt]: Why add a `lib` prefix?

Comment (seanyen) [reply recipe/CMakeLists.txt]: @isuruf Ah, I didn't really think of it is a problem at the first glance when I shamelessly get the recipe from `vcpkg`.  I removed the prefix now.

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
@conda-forge/jbig @conda-forge/staged-recipes Friendly ping. This is ready for review and merge. Thanks.

**Comment by scopatz** [comment]
@seanyen - this is a feedstock so please ping @conda-forge/jbig or @conda-forge/core if the maintainers are unresponsive. Thanks!
